front/src/components/BurgerMenu/SideMenu.tsx

```diff
@@ -1,16 +1,17 @@
-"use client";
+'use client';
 
-import { useAppContext } from "@/context/state";
-import { useLockBodyScroll } from "@/lib/useLockBodyScroll";
+import { useAppContext } from '@/context/state';
+import { useLockBodyScroll } from '@/lib/useLockBodyScroll';
 
-import clsx from "clsx";
-import classes from "./BurgerMenu.module.scss";
+import clsx from 'clsx';
+import classes from './BurgerMenu.module.scss';
 
-import { MenuLink } from "../Navbar";
-import { CustomButton } from "../Buttons/CustomButton";
+import { MenuLink } from '../Navbar';
+import { CustomButton } from '../Buttons/CustomButton';
 
-import { IMenuInclude } from "@/types/menu";
-import { email } from "@/types/contacts";
+import { IMenuInclude } from '@/types/menu';
```

Comment:
The best practice is to use "import type" when importing types or interfaces, it allows you to read the code better and, most importantly, optimizes runtime.

import type { IMenuInclude } from '@/types/menu';